src/Integration/LocalServices/SolutionBindingInformationProvider.cs

```diff
@@ -127,7 +127,9 @@ private RuleSet FindSonarQubeSolutionRuleSet(Dictionary<string, RuleSet> cache,
 
             string expectedSolutionRuleSet = ruleSetInfoProvider.CalculateSolutionSonarQubeRuleSetFilePath(
                          binding.ProjectKey,
-                         Language.ForProject(project));
+                         Language.ForProject(project),
+                         // TODO: duncanp decide whether to support new connected mode
```

Comment:
I've added a couple of specific reminders for myself of additional features to look at before shipping (I had quick look at a couple of the ruleset/conflict manager classes and couldn't see anything that was specific to NuGet so they might work ok with the new connected mode).